examples/columbia_plume/tidal_forcing.py

```diff
@@ -0,0 +1,98 @@
+import thetis.coordsys as coordsys
+import thetis.timezone as timezone
+import uptide
+import uptide.tidal_netcdf
+from firedrake import *
+import numpy as np
+import os
+
+tide_file = 'forcings/tide.fes2004.nc'
+msg = 'File {:} not found, download it from \nftp://ftp.legos.obs-mip.fr/pub/soa/maree/tide_model/global_solution/fes2004/'.format(tide_file)
+assert os.path.exists(tide_file), msg
+
+
+def to_latlon(x, y):
+    lon, lat = coordsys.convert_coords(coordsys.SPCS_N_OR, coordsys.LL_WGS84, x, y)
+    if lon < 0.0:
+        lon += 360.
+    return lat, lon
+
+
+class TidalBoundaryForcing(object):
```

Comment:
This looks like it could be useful functionality to include in Thetis - we have a similar "tidal_forcing.py" that gets copied around with all our different tidal setups. I can have a go at  moving this into a Thetis utility module  and generalize it (for coordinate systems and add the option of TPXO) after this merge?

Reply:
Yes, that sounds very good.